app/reducers/channels.js

```diff
@@ -290,9 +291,14 @@ const throttledFetchChannels = throttle(dispatch => dispatch(fetchChannels()), 1
 export const channelGraphData = (event, data) => (dispatch, getState) => {
   const { info, channels } = getState()
   const {
-    channelGraphData: { channel_updates }
+    channelGraphData: { channel_updates, node_updates }
   } = data
 
+  // Process node updates.
+  node_updates.forEach(nodeData => {
```

Comment:
It maybe outside of scope of this PR, but later we may want to enhance`updateNodeData` to accept multiple nodes to mutate state in one go to avoid potential extra re-renders in case data comes in bulk.

Reply:
Good idea, I'll have a look to see if this can be done easily. In my testing I have only ever see a single update come through at once, though given it is an array I assume it is possible for multiple updates to come through together.

Reply:
Have updated to handle multiple node updates in a single dispatch

Reply:
Looks good!